File: packages/cli/tests/commands/template/fixtures/copy-variations/expected/typescript/template.ts
import { Template } from 'e2b'

export const template = Template()
  .fromImage('alpine:latest')
  .setUser('root')
  .setWorkdir('/')
  .copy('package.json', '/app/')
  .copy('src/index.js', './src/')
  .copy('config.json', '/etc/app/config.json')
  .setUser('user')
  .setWorkdir('/home/user')

File: packages/cli/tests/commands/template/fixtures/copy-variations/expected/python-sync/template.py
from e2b import Template

template = (
    Template()
    .from_image("alpine:latest")
    .set_user("root")
    .set_workdir("/")
    .copy("package.json", "/app/")
    .copy("src/index.js", "./src/")
    .copy("config.json", "/etc/app/config.json")
    .set_user("user")
    .set_workdir("/home/user")
)